### PR TITLE
Consume CSC from chrome-service

### PR DESCRIPTION
**Pull request by Hyperkid123**
part of: https://issues.redhat.com/browse/RHCLOUD-25055

requires: https://github.com/RedHatInsights/chrome-service-backend/pull/33

### CODE REVIEW

File: src/utils/common.ts [hunk 1]
```diff
@@ -314,14 +314,48 @@ export const trustarcScriptSetup = () => {
...
+
+export const loadFedModules = async () => {
+  const commnHeaders = {
+    'Cache-Control': 'no-cache',
+    Pragma: 'no-cache',
+    Expires: '0',
+  };
+  // we need to fetch both old and new to get the chrome config from FEO
+  const [FEOModules, staticModules] = await Promise.all([
+    axios.get(`${window.location.origin}${isBeta() ? '/beta' : ''}/config/chrome/fed-modules.json?ts=${Date.now()}`, {
```
Comment (karelhala) [inline src/utils/common.ts:343]: We are not getting the fed-modules config from FEO. I don't think we need this thing.

**Review by karelhala — COMMENTED** [review]
Looking good! Just a minor question wrt getting fed-mods from FEO.

File: src/utils/common.ts (hunk 1 above)
Comment (Hyperkid123) [reply src/utils/common.ts:343]: I think we still need this for EE right? For the SSO URL and stuff. And also console.dev

Comment (karelhala) [reply src/utils/common.ts:343]: Aaah, yes! For the ephemeral environment we'll need this. For the console.dev not anymore.

**Review by karelhala — APPROVED** [review]

**Comment by Hyperkid123** [comment]
Turns out we can't use this yet because of 3scale auth gating. We need and exception.

**Comment by Hyperkid123** [comment]
OK I have sequenced the data loading to ensure the ssoURL gets loaded from legacy config in necessary environments and after auth was sucesfull we load the modules config from chrome-service. This should also add a for qaprodauth.

**Review by karelhala — APPROVED** [review]

**Pull request merged**